data/hai/hai culture conversations.txt

```diff
@@ -207,3 +207,47 @@ mission "Gambling Judgment"
 			label end
 			`	He takes your order diligently and with a smile.`
 				decline
+
+
+mission "Robotic Musical 1"
+	minor
+	source
+		government "Hai"
+		attributes "tourism"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Passing down the main thoroughfare of this spaceport you almost ignore a stack of haphazardly piled crates in the median strip. It's not uncommon for recently unloaded crates to temporarily sit in front of different businesses in a port, place conspicuously out of the way of the flow of foot traffic, but it is the disordered piling that seems out of place.`
+			`	No sooner do you slow your step to give it your attention though, than the pile of crates immediately spring to life.`
+			branch translator
+				has "language: Wanderer"
+			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions presumably intended to be comedic, at least judging by how they keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs, presumably advertisement information, flicker through a series of displays as it goes through a cycle before the entire cohort appear to finish up some kind of entreaty and immediately clamber back into the crates.`
```

Comment:
```suggestion
			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions presumably intended to be comedic, at least judging by how they keep turning away from each other to "look" at the audience and wink. Holoprojected banners and glyphs, presumably advertisement information, flicker through a series of displays as it goes through a cycle before the entire cohort appears to finish up some kind of entreaty and immediately clamber back into the crates.`
```

Reply:
Pretty sure "appear" is correct as this is used in the context of a group noun.

https://dictionary.cambridge.org/grammar/british-grammar/appear

Reply:
It's a [collective noun](https://www.touro.edu/departments/writing-center/tutorials/verb-agreement-with-collective-singular-nouns/). Basically, if the `cohort` acts as one collective group in "appearing to finish up", then you use singular verb (appears) instead of plural (appear). I suppose you could say that they're performing their action as individuals (in which case "appear" would be right), but it doesn't really feel like that to me, at least. If you'd prefer to keep it as-is though, I'm cool with letting it slide.
```suggestion
			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions presumably intended to be comedic, at least judging by how they keep turning away from each other to "look" at the audience and wink. Holoprojected banners and glyphs, presumably advertisement information, flicker through a series of displays as it goes through a cycle before the entire cohort appear to finish up some kind of entreaty and immediately clamber back into the crates.`
```

Reply:
Well that was an interesting exercise in re-wiring my brain. I think the implication I was going for is that they'd divide their attention among onlookers, meaning they all either individually or in small groups are performing separate entreaties at various onlookers. Which would make it "appear"; however, I haven't carried across the particulars of that sufficiently well to make it clear that that is what is happening.

Reply:
If that's the intent, then I'd agree that `appear` is right. I do think adding a sentence or phrase to clarify that this is performed as individuals rather than a group would be helpful, but I don't think it's absolutely necessary, especially for a culture mission. I'll leave it up to you to decide!